src/reminder.c

```diff
@@ -0,0 +1,139 @@
+char cur_date[10];
+char sc_date[10];
+int notification_count=0;
+
+//Checks if valid date is entered
+int valid_date(char * date)
+{
+int i =0;
+while(date[i]!='\0')
+{
+if((i==2||i==5)){
+if(date[i]!=47)
+return 0;
+}
+else if(date[i] < 48 || date[i] > 57){
+return 0;
+}
+i++;
+}
+return 1;
+}
+
+
+//gets the events of the current day if any
+int cur_schedule()
+{
+int events = 0;
+struct tm * cur_time;
+event search_event;
+time_t cur_timer;
+cur_timer = time(NULL);
+cur_time = localtime( &cur_timer );
+strftime(cur_date,10,"%x", cur_time);
+FILE * fp;
+fp = fopen("events.dat","r");
+if(fp == NULL)
+{
+printf("Error opening schedule\n");
+return 0;
+}
+while(fread(&search_event,sizeof(event),1,fp))
+{
+if(!strcmp(search_event.date,cur_date)){
+events += 1;
+}
+}
+fclose(fp);
+return events;
+}
+
+//displays schedule of date passed to function
+int show_schedule(char * sc_date)
+{
+int events;
+FILE * fp;
+event search_event;
+fp = fopen("events.dat","r");
+if(fp == NULL)
+{
+printf("Error opening schedule\n");
+return 0;
+}
+while(fread(&search_event,sizeof(event),1,fp))
+{
+if(!strcmp(search_event.date,sc_date)){
+printf("You have %s on %s at %s", search_event.desc,search_event.date,search_event.time);
```

Comment:
```suggestion
printf("You have %s on %s at %s \n", search_event.desc,search_event.date,search_event.time);
```

You missed a \n at the end here.